src/Uno.Extensions.Templates/content/unoapp-extensions/Directory.Build.nocpm.props

```diff
@@ -14,6 +14,6 @@
 
 	<ItemGroup>
 		<!--This override is used to validate the use of specific version of the C# Compiler-->
-		<PackageReference Include="Microsoft.Net.Compilers.Toolset" Version="4.2.0"  PrivateAssets="all" />
+		<PackageReference Include="Microsoft.Net.Compilers.Toolset" Version="4.4.0"  PrivateAssets="all" />
```

Comment:
This version is known to not be available on some build agent (mac when building xamarin project if I'm right). They will fallback to a lower version, but if you are using a specific feature of this version, the build will fail.